src/main/java/com/mg105/user_interface/TutorialTextWindow.java

```diff
@@ -0,0 +1,101 @@
+package com.mg105.user_interface;
+
+import com.mg105.controllers.TutorialTextController;
+import com.mg105.utils.TutorialTexts;
+import javafx.animation.AnimationTimer;
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.Pane;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.stage.Stage;
+
+public class TutorialTextWindow extends Application {
+    TutorialTextDisplay tutorialDisplay = new TutorialTextDisplay();
+    Label bottomText = new Label();
+    Label helpText = new Label();
+    TutorialTextController textController;
+
+    Pane tutorialPane = new Pane();
+
+    public TutorialTextWindow(TutorialTextController textController) {
+        this.textController = textController;
+    }
+
+
+    public static void main(String[] args) {
+        launch(args);
+    }
+
+    @Override
+    public void start(Stage tutorialStage) {
+        bottomText = tutorialDisplay.tutorialLabel();
+        bottomText.setFont(Font.font(TutorialTexts.textFont1, TutorialTexts.textSize));
+
+        helpText = tutorialDisplay.tutorialLabel();
+        helpText.setFont(Font.font(TutorialTexts.textFont2, TutorialTexts.textSize));
+        helpText.setBackground(new Background(new BackgroundFill(Color.rgb(255, 215, 0, 1),
+            new CornerRadii(30.0), new Insets(-40.0))));
+
+        tutorialPane.getChildren().add(bottomText);
+        tutorialPane.getChildren().add(helpText);
+
+        Scene tutorialScene = new Scene(tutorialPane, TutorialTexts.helperPaneX, TutorialTexts.helperPaneY);
+
+        AnimationTimer timer = new TutorialTimer();
+        timer.start();
+
+        tutorialStage.setScene(tutorialScene);
+        tutorialStage.show();
+    }
```

Comment:
Instead of creating a new window, it should just modify the current window via the currently existing `Toggler` and `Toggleable` interface.

This would also have the effect of removing the edge case where the user closes the side window.